src/messages.ts

```diff
@@ -1,7 +1,7 @@
 import chalk from "chalk"
 import type { Preferences } from "./prompts"
-import { getUserPkgManager } from "./utils/getUserPkgManager"
 import { getVersion } from "./utils/getVersion"
+import { getUserPkgManager } from "./utils/getUserPkgManager"
```

Comment:
This change seems to be unnecessary, is there a reason you swapped the imports ?